listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -171,12 +171,8 @@ def make_api_request(user, spotipy_call, **kwargs):
                 if not tried_to_refresh_token:
                     try:
                         user = spotify.refresh_user_token(user)
-                    except SpotifyException as err:
+                    except (SpotifyException, spotify.SpotifyInvalidGrantError) as err:
```

Comment:
This feels a bit funny catching both an exception from Spotify and an exception raised from `refresh_user_token` in the same line. Do we think that the error message in both of these cases should be the same?
In the case where we know (?) that the user has removed permission, we report it as "please unlink and link your account again". Perhaps in this case we can explicitly say something like "It seems that we no longer have permission to read your account"

Reply:
Now that I go through this part again, I don't think we need to catch these exceptions here. These will be handled in `process_one_user`.